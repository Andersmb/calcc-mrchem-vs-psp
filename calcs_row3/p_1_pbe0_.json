{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -1.789003595
          ],
          "radius": 1.23
        },
        {
          "center": [
            0.0,
            0.0,
            1.789003595
          ],
          "radius": 1.23
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "p",
          "xyz": [
            0.0,
            0.0,
            -1.789003595
          ]
        },
        {
          "atom": "p",
          "xyz": [
            0.0,
            0.0,
            1.789003595
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "p_1_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -3.54000363932091e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 1.299844690018193e-05,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -6.080864505232403e-08,
            -2.940607685430111e-08,
            -1.2998271400774033e-05
          ],
          "vector_el": [
            -6.097585414366335e-08,
            -2.9573217652512085e-08,
            -1.2998271446959311e-05
          ],
          "vector_nuc": [
            1.6720909133933333e-10,
            1.6714079821097713e-10,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "P",
          "xyz": [
            0.0,
            0.0,
            -1.789003595
          ]
        },
        {
          "symbol": "P",
          "xyz": [
            0.0,
            0.0,
            1.789003595
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -77.23459252564336,
          -77.23458929923677,
          -6.633996402993258,
          -6.633480546100714,
          -4.769243570385835,
          -4.769243568322475,
          -4.769034456216825,
          -4.769034455010737,
          -4.766192805460482,
          -4.763969584178192,
          -0.7003745135241398,
          -0.4790123827300695,
          -0.3079620554882049,
          -0.2980195009101743,
          -0.2980194966166914,
          -77.23459252770716,
          -77.23458930342403,
          -6.633996410722978,
          -6.63348055396645,
          -4.769243574243478,
          -4.769243572278261,
          -4.769034460330553,
          -4.769034459065539,
          -4.766192829474137,
          -4.7639696082444045,
          -0.700374513694854,
          -0.4790123894874069,
          -0.30796205550246664,
          -0.2980195022554451,
          -0.29801949939553507
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -396.8535304226106
      },
      "scf_energy": {
        "E_ee": 369.98199151543713,
        "E_eext": 0.0,
        "E_el": -745.3546714873096,
        "E_en": -1750.6380274194046,
        "E_kin": 681.4610923700059,
        "E_next": 0.0,
        "E_nn": 62.88416653517121,
        "E_nuc": 62.88416653517121,
        "E_tot": -682.4705049521384,
        "E_x": -11.293385544860033,
        "E_xc": -34.86634240848801,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 368.9895960388219,
        "E_eext": 0.0,
        "E_el": -741.8075670381772,
        "E_en": -1741.4339258247317,
        "E_kin": 676.6921989834026,
        "E_next": 0.0,
        "E_nn": 62.88416653517121,
        "E_nuc": 62.88416653517121,
        "E_tot": -678.9234005030061,
        "E_x": -11.270953460022849,
        "E_xc": -34.78448277564712,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 369.0430480669729,
              "E_eext": 0.0,
              "E_el": -745.2281586170369,
              "E_en": -1743.3511741416996,
              "E_kin": 675.1121069605398,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.3439920818657,
              "E_x": -11.26461066483588,
              "E_xc": -34.76752883801395,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.3439920818657,
            "energy_update": 3.4205915788596712,
            "mo_residual": 0.21586400234197864,
            "wall_time": 151.532794442
          },
          {
            "energy_terms": {
              "E_ee": 369.5063110248646,
              "E_eext": 0.0,
              "E_el": -745.3319825211628,
              "E_en": -1747.2851425087513,
              "E_kin": 678.5399622664316,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4478159859916,
              "E_x": -11.277739904273856,
              "E_xc": -34.81537339943371,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4478159859916,
            "energy_update": 0.10382390412587483,
            "mo_residual": 0.06676476230996166,
            "wall_time": 102.245162072
          },
          {
            "energy_terms": {
              "E_ee": 369.7106547074074,
              "E_eext": 0.0,
              "E_el": -745.3479273612371,
              "E_en": -1749.0822334564584,
              "E_kin": 680.1484926017632,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4637608260659,
              "E_x": -11.284748337451502,
              "E_xc": -34.840092876497856,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4637608260659,
            "energy_update": 0.01594484007432584,
            "mo_residual": 0.03409531625964039,
            "wall_time": 88.946088707
          },
          {
            "energy_terms": {
              "E_ee": 369.9846719611506,
              "E_eext": 0.0,
              "E_el": -745.3532225369628,
              "E_en": -1750.8010053199782,
              "E_kin": 681.6244500745833,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4690560017916,
              "E_x": -11.2932233808352,
              "E_xc": -34.86811587188318,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4690560017916,
            "energy_update": 0.0052951757256778365,
            "mo_residual": 0.04021273856220262,
            "wall_time": 94.542128571
          },
          {
            "energy_terms": {
              "E_ee": 370.02851658934117,
              "E_eext": 0.0,
              "E_el": -745.3542785074076,
              "E_en": -1751.0835187917635,
              "E_kin": 681.8683504257041,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4701119722364,
              "E_x": -11.294824633258676,
              "E_xc": -34.872802097430565,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4701119722364,
            "energy_update": 0.001055970444781451,
            "mo_residual": 0.0168631389412988,
            "wall_time": 92.903620011
          },
          {
            "energy_terms": {
              "E_ee": 370.0024481917585,
              "E_eext": 0.0,
              "E_el": -745.354607401188,
              "E_en": -1750.7689224309254,
              "E_kin": 681.5742982577706,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4704408660168,
              "E_x": -11.293857304909002,
              "E_xc": -34.86857411488271,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4704408660168,
            "energy_update": 0.00032889378042000317,
            "mo_residual": 0.010661869710668206,
            "wall_time": 90.509642731
          },
          {
            "energy_terms": {
              "E_ee": 369.99207914162685,
              "E_eext": 0.0,
              "E_el": -745.3546576539756,
              "E_en": -1750.6460012999273,
              "E_kin": 681.4597689121208,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4704911188044,
              "E_x": -11.293542524687025,
              "E_xc": -34.866961883108814,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4704911188044,
            "energy_update": 5.025278755965701e-05,
            "mo_residual": 0.00287040519167371,
            "wall_time": 95.484944436
          },
          {
            "energy_terms": {
              "E_ee": 369.9822321384557,
              "E_eext": 0.0,
              "E_el": -745.3546697226358,
              "E_en": -1750.6167527427644,
              "E_kin": 681.4392596683667,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4705031874646,
              "E_x": -11.293305419175185,
              "E_xc": -34.86610336751844,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4705031874646,
            "energy_update": 1.206866022585018e-05,
            "mo_residual": 0.0013012782054665556,
            "wall_time": 96.639504199
          },
          {
            "energy_terms": {
              "E_ee": 369.98024418720723,
              "E_eext": 0.0,
              "E_el": -745.3546709941567,
              "E_en": -1750.6216089216307,
              "E_kin": 681.4460430372965,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4705044589855,
              "E_x": -11.293291638858463,
              "E_xc": -34.86605765817127,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4705044589855,
            "energy_update": 1.2715208868030459e-06,
            "mo_residual": 0.000440462562905031,
            "wall_time": 94.963955697
          },
          {
            "energy_terms": {
              "E_ee": 369.9813299104853,
              "E_eext": 0.0,
              "E_el": -745.3546714076529,
              "E_en": -1750.6340345039168,
              "E_kin": 681.4576637939826,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4705048724817,
              "E_x": -11.2933634283547,
              "E_xc": -34.86626717984928,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4705048724817,
            "energy_update": 4.1349619550601346e-07,
            "mo_residual": 0.00028623088196657367,
            "wall_time": 98.030885229
          },
          {
            "energy_terms": {
              "E_ee": 369.98199151543713,
              "E_eext": 0.0,
              "E_el": -745.3546714873096,
              "E_en": -1750.6380274194046,
              "E_kin": 681.4610923700059,
              "E_next": 0.0,
              "E_nn": 62.88416653517121,
              "E_nuc": 62.88416653517121,
              "E_tot": -682.4705049521384,
              "E_x": -11.293385544860033,
              "E_xc": -34.86634240848801,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -682.4705049521384,
            "energy_update": 7.965672921272926e-08,
            "mo_residual": 0.00012322632605792463,
            "wall_time": 113.351135687
          }
        ],
        "wall_time": 1119.228676707
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
