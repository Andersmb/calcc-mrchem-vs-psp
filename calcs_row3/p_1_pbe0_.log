Starting job 5688129 on c1-[21,26],c3-[13,27],c5-22,c10-5 at Fri May 20 04:35:53 CEST 2022

Copying p_1_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying p_1_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688129       p_1_pbe0_        180                                             00:20:46      0:0 
5688129.bat+      batch         24        1   00:00:03          0   00:00:03   00:20:46      0:0 
5688129.ext+     extern        180        6   00:00:00          5   00:00:00   00:20:46      0:0 
5688129.0      mrchem.x        180       30   00:20:14         29   00:55:32   00:20:24      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688129                                                                          
5688129.bat+     19344K          0     19344K        0              0          0 
5688129.ext+          0          5          0        0              5          0 
5688129.0     10464044K         29 3893677260        0             11          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688129                                                                                               
5688129.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688129.ext+        0.00M               5          0.00M            0                5              0 
5688129.0           7.32M              28          6.81M      121.93M                0         55.56M 

Job 5688129 completed at Fri May 20 04:56:39 CEST 2022
