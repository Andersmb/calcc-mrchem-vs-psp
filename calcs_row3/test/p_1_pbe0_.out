                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -1.789004
    1       P            :        0.000000        0.000000        1.789004
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              15
 Beta electrons          :                                              15
 Total electrons         :                                              30
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000096100e+00
    1    1    a          :                              1.000000773335e+00
    2    1    a          :                              1.000002119446e+00
    3    1    a          :                              9.999909910156e-01
    4    1    a          :                              9.999834610489e-01
    5    1    a          :                              1.000016638812e+00
    6    1    a          :                              1.000001006038e+00
    7    1    a          :                              1.000001059141e+00
    8    1    a          :                              9.999989471705e-01
    9    1    a          :                              9.999989471705e-01
   10    1    a          :                              9.999988229777e-01
   11    1    a          :                              1.000000106638e+00
   12    1    a          :                              1.000000170429e+00
   13    1    a          :                              9.999999285948e-01
   14    1    a          :                              9.999999285948e-01
   15    1    b          :                              1.000000338904e+00
   16    1    b          :                              9.999995827310e-01
   17    1    b          :                              1.000002119259e+00
   18    1    b          :                              9.999978851222e-01
   19    1    b          :                              9.999835685283e-01
   20    1    b          :                              1.000016514066e+00
   21    1    b          :                              1.000001058500e+00
   22    1    b          :                              1.000001058500e+00
   23    1    b          :                              9.999989539331e-01
   24    1    b          :                              9.999987251086e-01
   25    1    b          :                              9.999988229776e-01
   26    1    b          :                              1.000000105318e+00
   27    1    b          :                              1.000000167487e+00
   28    1    b          :                              9.999999285948e-01
   29    1    b          :                              9.999999285948e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                676.692198983403
 E-N energy              :            (au)              -1741.433925824732
 Coulomb energy          :            (au)                368.989596038822
 Exchange energy         :            (au)                -11.270953460023
 X-C energy              :            (au)                -34.784482775647
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 62.884166535171
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -741.807567038177
 Nuclear energy          :            (au)                 62.884166535171
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.789234005030e+02
                         :      (kcal/mol)             -4.260308625413e+05
                         :        (kJ/mol)             -1.782513130030e+06
                         :            (eV)             -1.847444607054e+04
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -76.711257235545
    1    1    a          :            (au)                -76.704657505312
    2    1    a          :            (au)                 -6.599126502647
    3    1    a          :            (au)                 -6.594586950340
    4    1    a          :            (au)                 -4.734893669577
    5    1    a          :            (au)                 -4.734888571633
    6    1    a          :            (au)                 -4.733858972902
    7    1    a          :            (au)                 -4.730795108640
    8    1    a          :            (au)                 -4.730794694528
    9    1    a          :            (au)                 -4.729179196185
   10    1    a          :            (au)                 -0.742536627741
   11    1    a          :            (au)                 -0.466958224563
   12    1    a          :            (au)                 -0.307908712037
   13    1    a          :            (au)                 -0.307906183940
   14    1    a          :            (au)                 -0.297845703325
   15    1    b          :            (au)                -76.711295959672
   16    1    b          :            (au)                -76.704627972376
   17    1    b          :            (au)                 -6.599082598673
   18    1    b          :            (au)                 -6.594631260800
   19    1    b          :            (au)                 -4.734890666142
   20    1    b          :            (au)                 -4.734888629748
   21    1    b          :            (au)                 -4.733850713799
   22    1    b          :            (au)                 -4.730797026617
   23    1    b          :            (au)                 -4.730794724406
   24    1    b          :            (au)                 -4.729175965190
   25    1    b          :            (au)                 -0.742561456503
   26    1    b          :            (au)                 -0.466953630383
   27    1    b          :            (au)                 -0.307917616453
   28    1    b          :            (au)                 -0.307916899840
   29    1    b          :            (au)                 -0.297850225247
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -394.254429204765
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          5.477226e+00        -678.923400503006         -6.789234e+02
    1          2.158640e-01        -682.343992081866         -3.420592e+00
    2          6.676476e-02        -682.447815985992         -1.038239e-01
    3          3.409532e-02        -682.463760826066         -1.594484e-02
    4          4.021274e-02        -682.469056001792         -5.295176e-03
    5          1.686314e-02        -682.470111972236         -1.055970e-03
    6          1.066187e-02        -682.470440866017         -3.288938e-04
    7          2.870405e-03        -682.470491118804         -5.025279e-05
    8          1.301278e-03        -682.470503187465         -1.206866e-05
    9          4.404626e-04        -682.470504458985         -1.271521e-06
   10          2.862309e-04        -682.470504872482         -4.134962e-07
   11          1.232263e-04        -682.470504952138         -7.965673e-08
---------------------------------------------------------------------------
                      SCF converged in 11 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       P            :        0.000000        0.000000       -1.789004
    1       P            :        0.000000        0.000000        1.789004
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                681.461092370006
 E-N energy              :            (au)              -1750.638027419405
 Coulomb energy          :            (au)                369.981991515437
 Exchange energy         :            (au)                -11.293385544860
 X-C energy              :            (au)                -34.866342408488
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 62.884166535171
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -745.354671487310
 Nuclear energy          :            (au)                 62.884166535171
---------------------------------------------------------------------------
 Total energy            :            (au)             -6.824705049521e+02
                         :      (kcal/mol)             -4.282567041707e+05
                         :        (kJ/mol)             -1.791826051413e+06
                         :            (eV)             -1.857096769552e+04
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -77.234592525643
    1    1    a          :            (au)                -77.234589299237
    2    1    a          :            (au)                 -6.633996402993
    3    1    a          :            (au)                 -6.633480546101
    4    1    a          :            (au)                 -4.769243570386
    5    1    a          :            (au)                 -4.769243568322
    6    1    a          :            (au)                 -4.769034456217
    7    1    a          :            (au)                 -4.769034455011
    8    1    a          :            (au)                 -4.766192805460
    9    1    a          :            (au)                 -4.763969584178
   10    1    a          :            (au)                 -0.700374513524
   11    1    a          :            (au)                 -0.479012382730
   12    1    a          :            (au)                 -0.307962055488
   13    1    a          :            (au)                 -0.298019500910
   14    1    a          :            (au)                 -0.298019496617
   15    1    b          :            (au)                -77.234592527707
   16    1    b          :            (au)                -77.234589303424
   17    1    b          :            (au)                 -6.633996410723
   18    1    b          :            (au)                 -6.633480553966
   19    1    b          :            (au)                 -4.769243574243
   20    1    b          :            (au)                 -4.769243572278
   21    1    b          :            (au)                 -4.769034460331
   22    1    b          :            (au)                 -4.769034459066
   23    1    b          :            (au)                 -4.766192829474
   24    1    b          :            (au)                 -4.763969608244
   25    1    b          :            (au)                 -0.700374513695
   26    1    b          :            (au)                 -0.479012389487
   27    1    b          :            (au)                 -0.307962055502
   28    1    b          :            (au)                 -0.298019502255
   29    1    b          :            (au)                 -0.298019499396
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -396.853530422611
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -0.000013
 Magnitude               :            (au)                        0.000013
                         :         (Debye)                        0.000033
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000       -0.000000       -0.000013
 Magnitude               :            (au)                        0.000013
                         :         (Debye)                        0.000033
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 20m 14s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
